spec/factories/user_action_events.rb

```diff
@@ -3,5 +3,7 @@
 FactoryBot.define do
   factory :user_action_event do
     details { Faker::Lorem.sentence }
+    event_id { SecureRandom.uuid }
```

Comment:
Apologies, at some point I got my signals crossed and thought that this was an actual uuid instead of the `slug` it was originally named. We should probably update this factory definition to `'some_event_id'` or something similar so that its purpose is clear.